src/kOS/Screen/KOSToolBarWindow.cs

```diff
@@ -56,7 +54,7 @@ public class KOSToolBarWindow : MonoBehaviour
         // ReSharper restore RedundantDefaultFieldInitializer
         private Vector2 scrollPos = new Vector2(200,350);
 
-        private static Rect windowRect;
+        private static Rect windowRect; // does anybody know why this is ststic?
```

Comment:
But some per-window fields (e.g. isOpen, clickedOn, width, height) are not static.